source/tutorials/kubernetes.rst

```diff
@@ -275,77 +525,91 @@ If you have already set your proxy environment variables, run the following comm
 
 * <HOSTNAME> not found in <IP> message.
 
-  Your DNS server may not be appropriately configured. Try adding an entry to the :file:`/etc/hosts` file with your host's IP and Name.
+   Your DNS server may not be appropriately configured. Try adding an entry
+   to the :file:`/etc/hosts` file with your host's IP and Name.
 
-  For example: 100.200.50.20 myhost
+   For example: 100.200.50.20 myhost
 
-  Use the commands :command:`hostname` and :command:`hostname -I` to retrieve them.
+   Use the commands :command:`hostname` and :command:`hostname -I` to
+   retrieve them.
 
 * Images cannot be pulled.
 
-  You may be behind a proxy server. Try configuring your proxy settings, using the environment variables *HTTP_PROXY*, *HTTPS_PROXY*, and *NO_PROXY* as required in your environment.
+   You may be behind a proxy server. Try configuring your proxy settings,
+   using the environment variables *HTTP_PROXY*, *HTTPS_PROXY*, and
+   *NO_PROXY* as required in your environment.
 
 * Connection refused error.
 
-  If you are behind a proxy server, you may need to add the master's IP to the environment variable *NO_PROXY*.
+   If you are behind a proxy server, you may need to add the master's IP to
+   the environment variable *NO_PROXY*.
 
 * Connection timed-out or Access Refused errors.
 
-  You must ensure that the appropriate proxy settings are available from the same terminal where you will initialize the control plane. To verify the proxy settings that Kubernetes will actually use, run the commands:
+   You must ensure that the appropriate proxy settings are available from the
+   same terminal where you will initialize the control plane. To verify the
+   proxy settings that Kubernetes will actually use, run the commands:
 
-  .. code-block:: bash
+   .. code-block:: bash
 
-     echo $HTTP_PROXY
-     echo $HTTPS_PROXY
-     echo $NO_PROXY
+      echo $HTTP_PROXY
+      echo $HTTPS_PROXY
+      echo $NO_PROXY
 
-  If the displayed proxy values are different from your assigned values, the cluster initialization will fail. Contact your IT support team to learn how to set the proxy variables permanently, and how to make them available for all the types of access that you will use, such as remote SSH access.
+   If the displayed proxy values are different from your assigned values, the
+   cluster initialization will fail. Contact your IT support team to learn how
+   to set the proxy variables permanently, and how to make them available for
+   all the types of access that you will use, such as remote SSH access.
 
-  If the result of the above commands is blank, you may need to add a ``profile`` to the :file:`/etc` directory. To do so, follow these steps.
+   If the result of the above commands is blank, you may need to add a
+   ``profile`` to the :file:`/etc` directory to set environment variables
+   persistently. To do so, follow these steps.
 
-  #. Create a `profile` in :file:`/etc`
+   #. Create a `profile` in :file:`/etc`
```

Comment:
These set of instructions are confusing.  I think you meant to say `cd /etc` first before running `sudo touch profile`  The `cloud-native-setup` [script](https://github.com/clearlinux/cloud-native-setup/blob/00df885b45322a1d3b56562fef908f1c960b4690/clr-k8s-examples/setup_system.sh#L153) checks `/etc/profile.d` directory for proxy environment variables. That is probably a better directory to put this information in.

Reply:
This was existing content that I didn't check (the whole troubleshooting section). I'll update it in anycase but we can decide how to revise or pull out this section as a whole later.